pkgs/top-level/python-packages.nix

```diff
@@ -13940,6 +13940,23 @@ in {
     };
   });
 
+  ltc_scrypt = buildPythonPackage rec {
```

Comment:
Should it be moved because of the following?

> Python packages that do not need to be available for each interpreter version do not belong in this packages set.

If so, where should it be moved to?